src/instruments/src/EFB/Ground/Pages/Payload/Payload.tsx

```diff
@@ -329,6 +329,34 @@ export const Payload = () => {
         ...desiredFlags, ...stationSize,
     ]);
 
+    // TODO: Rework A32NX_Boarding to make boardingRateMultiplier obsolete. (msDelay)
+    const calculateBoardingTime = () => {
+        let estimatedTimeSeconds = 0;
+        let estimatedPaxBoardingSeconds = 0;
+        let estimatedCargoLoadingSeconds = 0;
+        const boardingRateMultiplier = 5;
+        const differentialPax = Math.abs(totalPaxDesired - totalPax);
+        const differentialCargo = Math.abs(totalCargoDesired - totalCargo);
+
+        if (boardingRate === 'REAL') {
+            estimatedPaxBoardingSeconds += differentialPax * boardingRateMultiplier;
+            estimatedCargoLoadingSeconds += (differentialCargo / 60) * boardingRateMultiplier;
+        } else if (boardingRate === 'FAST') {
+            estimatedPaxBoardingSeconds += differentialPax;
+            estimatedCargoLoadingSeconds += (differentialCargo / 60);
+        }
+
+        if (estimatedPaxBoardingSeconds > estimatedCargoLoadingSeconds) {
+            const differentialLoadingTime = Math.abs(estimatedPaxBoardingSeconds - estimatedCargoLoadingSeconds);
+            estimatedTimeSeconds = estimatedPaxBoardingSeconds + differentialLoadingTime;
+        } else {
+            const differentialLoadingTime = Math.abs(estimatedCargoLoadingSeconds - estimatedPaxBoardingSeconds);
+            estimatedTimeSeconds = estimatedCargoLoadingSeconds + differentialLoadingTime;
+        }
+
+        return ` ${Math.round(estimatedTimeSeconds / 60)}`;
```

Comment:
return a number and convert in the HTML output
```suggestion
        return Math.round(estimatedTimeSeconds / 60);
```